api/pkg/transformer/types/series/series.go

```diff
@@ -12,10 +12,14 @@ import (
 type Type string
 
 const (
-	String Type = "string"
-	Int    Type = "int"
-	Float  Type = "float"
-	Bool   Type = "bool"
+	String     Type = "string"
```

Comment:
can you use valueType from common.pb.go instead of this? so we can remove this, and have one single truth of list of types

Reply:
I put it as out of scope for now as it introduce many changes at many places. Currently we are using string data type to define new series, however our common.pb.go's valueType is int32, hence we need to modify the signature of our merlin's series.New().

Reply:
ok